checksec

```diff
@@ -764,6 +763,14 @@ kernelcheck() {
     echo_message "\033[31mNone\033[m\n" "None," " randomize_va_space='none'" '"randomize_va_space":"none",'
   fi
 
+  echo_message "  NX protection:                          " "" "" ""
+  nx_protection=$(dmesg -t | grep -Fw NX)
```

Comment:
reading dmesg could be disallowed for unrpiv users. Should this case be handled there?